pkg/proxy/apis/kubeproxyconfig/validation/validation.go

```diff
@@ -238,3 +240,30 @@ func validateIPVSSchedulerMethod(scheduler kubeproxyconfig.IPVSSchedulerMethod,
 	}
 	return allErrs
 }
+
+func validateKubeProxyNodePortAddress(nodePortAddresses []string, fldPath *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+LOOP:
```

Comment:
you don't need this label?  `continue` does what you want, unless I am misunderstanding

Reply:
Yes, you are right. Label is unnecessary. That's a legacy code...